spec/takeaway_spec.rb

```diff
@@ -0,0 +1,41 @@
+require "takeaway"
+
+describe Takeaway do
+
+
+  let(:dishes){ {
```

Comment:
Style/BlockDelimiters: Avoid using `{...}` for multi-line blocks. (https://github.com/bbatsov/ruby-style-guide#single-line-blocks)<br>Style/MultilineBlockLayout: Block body expression is on the same line as the block start.